client/src/components/Subscribers/index.tsx

```diff
@@ -101,26 +104,31 @@ export class Subscribers extends React.Component<ISubscribersProps> {
         return (
             <Container>
                 <Menu/>
-                <div className='d-flex justify-content-center'>
-                    <Link to={`/profile/${this.props.urlUsername}/followers`} className='link-style'
-                          style={{textDecoration: !subscribers ? 'underline' : 'none'}}
-                    >
-                        <h4
-                            className='font-weight-light text-secondary text-uppercase'
+                {this.props.loading ?
+                        <div className='d-flex justify-content-center'>
+                            <Spinner className='mt-3' color='white'/>
+                        </div> :
+                        <div className='d-flex justify-content-center'>
+                        <Link to={`/profile/${this.props.urlUsername}/followers`} className='link-style'
+                              style={{textDecoration: !subscribers ? 'underline' : 'none'}}
                         >
-                            {this.props.followersCount} followers
-                        </h4>
-                    </Link>
-                    < Link to={`/profile/${this.props.urlUsername}/following`} className='link-style'
-                           style={{textDecoration: subscribers ? 'underline' : 'none'}}
-                    >
-                        <h4
-                            className='font-weight-light text-secondary text-uppercase'
+                            <h4
+                                className='font-weight-light text-secondary text-uppercase'
+                            >
+                                {this.props.followersCount} followers
+                            </h4>
+                        </Link>
```

Comment:
the same as in string 121--129. Move to function with argument for followers (or following) and reuse

Reply:
fixed